docs/source/development/style-guide/review-standards.rst

```diff
@@ -0,0 +1,65 @@
+the proposed changes are required and up to our standards.
+
+The PR should be as small as possible. However, the code included in the PR should be complete. It should complete an entire feature, but this doesn't necessarily mean it completes an entire issue.
+However, all code merged into the ``dev`` branch should be up to our standards and work sufficiently.
+
+If you want to work on the pull request or are not yet finished with the code, please indicate this by marking the pull request as a
+`draft or WIP <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests>`_ PR.
+Anyone looking at the PR will be able to quickly see it is not yet ready for review.
+
+Finally, if you are addressing an existing issue, make sure that issue is linked in your PR. If there is not an existing issue, then you should either create an issue or address WHY you are opening the PR specifically.
+If your PR addresses a `Level-5 requirement <https://github.com/orgs/IMAP-Science-Operations-Center/projects/2/views/2>`_, there **must** be a corresponding issue linked.
+
+During review
+=============
+
+As a reviewer, please follow these rules of thumb:
+
+#. Comments should be clear in addressing why you want to see the change
+#. Comments should be polite, but straightforward and candid
+#. If you leave a review, continue to follow up on replies to your questions or comments and review the changes you requested
+#. It is polite, but not required, to provide examples for suggestions (particularly for things like name changes)
+#. If you require a change to be addressed, add a "request changes" comment. If you make one of these comments, it means you are blocking the code review from merging until that change is addressed.
+#. If you make a "request changes" comment, you must create a follow up review where you change that to an approving review. Please do this in a timely matter so you do not block the PR for longer than necessary
+
+As an author:
+
+#. If you would like to request a specific review from someone, make sure they are marked as a reviewer or called out in a comment on the review (by typing ``@<username>``)
+#. You can request a review from the entire team or from a specific instrument team using the team ``IMAP-Science-Operations-Center/imap-sdc``
+
+Before merging
+==============
+
+Before merging, a pull request needs one approving review. While the review is open, anyone can make comments or request changes on the PR.
+
+Although only one approval is required, you must follow these rules:
+
+#. If there is someone with a particular expertise or vested interest in your changes, **do not merge or close the pull request until they get a chance to review.**
+#. Do not merge until you have addressed all the comments on your review, even if you have an approval from someone.
```

Comment:
This brings up two thoughts.
1. Are we approving too easily right now? Maybe we should save approvals for when we feel like it has addressed all of our comments?

2. Should we change from self-merging to reviewer-merging. i.e. we don't merge our own code, only a reviewer that thinks it is satisfactory would do that?

Reply:
I think leaving it as self-merging is fine. A reviewer might not know that, for example, I really want a review from someone specific but haven't gotten it yet.

What I was trying to express with this comment is that if Matthew gives me an approval, but Greg has outstanding comments, I should  not merge until I have addressed Greg's comments. 

I usually give an approval as a "when you fix this" type thing. If it's a minor change that I trust the author to make without needing another review, than I will give an approval, but I still want them to make the changes I suggested. In my opinion, even if the PR has approving reviews, the comments should still be addressed (where addressed means either fixed or replied to with why they will not be fixed). 

Do you or others have other opinions? I would worry about dragging out the review process for minor things - I think people won't approve something which they have larger concerns around. But if it's something minor, like suggesting additional documentation, then the reviewer doesn't necessarily need to go back and review a change after it happens. (e.g. if I say "please add a docstring here" then my approval stands for whatever docstring the author adds)

Reply:
> I usually give an approval as a "when you fix this" type thing. If it's a minor change that I trust the author to make without needing another review, than I will give an approval, but I still want them to make the changes I suggested. In my opinion, even if the PR has approving reviews, the comments should still be addressed (where addressed means either fixed or replied to with why they will not be fixed).

I usually do the same, so I agree with this. This was mostly just a thought of "if we care enough about these things", but it sounds like possibly not, so lets let it organically evolve and only update this if we run into issues in the future.